### PR TITLE
skip broken pandas test when on python 3.5

### PR DESCRIPTION
**Pull request by adnelson**
Pandas was failing to build with python 3.5; I searched around and found an [open ticket for pandas `0.17.0`](https://github.com/pydata/pandas/issues/11112). This change fixes builds on my machine (14.12.732.bc4f185 (Caterpillar))

@7c6f434c @FRidh

### CODE REVIEW

**Comment by domenkozar** [comment]
```
/bin/sh: 1: /usr/lib//tcl8.6/tclConfig.sh: dpkg-architecture: not found
             gtk3cairo: no  [Requires cairocffi or pycairo to be installed.]
                gtkagg: no  [Requires pygtk]
Traceback (most recent call last):
  File "setup.py", line 271, in <module>
    result = package.check()
  File "/tmp/nix-build-python3.5-matplotlib-1.5.0.drv-0/matplotlib-1.5.0/setupext.py", line 520, in check
    additional_info = self.check_requirements()
  File "/tmp/nix-build-python3.5-matplotlib-1.5.0.drv-0/matplotlib-1.5.0/setupext.py", line 1319, in check_requirements
    ext = self.get_extension()
  File "/tmp/nix-build-python3.5-matplotlib-1.5.0.drv-0/matplotlib-1.5.0/setupext.py", line 1339, in get_extension
    self.add_flags(ext)
  File "/tmp/nix-build-python3.5-matplotlib-1.5.0.drv-0/matplotlib-1.5.0/setupext.py", line 1576, in add_flags
    result = self.parse_tcl_config(tcl_lib_dir, tk_lib_dir)
  File "/tmp/nix-build-python3.5-matplotlib-1.5.0.drv-0/matplotlib-1.5.0/setupext.py", line 1432, in parse_tcl_config
    tcl_config, 'TCL_LIB_SPEC').split()[0][2:].strip()
IndexError: list index out of range
```

**Comment by adnelson** [comment]
Oh blargh. What OS was that on?

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -11850,12 +11851,19 @@ let
...
+                     "test_frequencies" "test_frame"
+                     "test_read_clipboard_infer_excel"] ++
+                    optional isPy35 "test_sql";
+    in ''
       runHook preCheck
       # The flag `-A 'not network'` will disable tests that use internet.
       # The `-e` flag disables a few problematic tests.
       ${python.executable} setup.py nosetests -A 'not network' --stop \
-        -e 'test_data|test_excel|test_html|test_json|test_frequencies|test_frame|test_read_clipboard_infer_excel' --verbosity=3
+        -e '${concatStringsSep "|" testsToSkip}' --verbosity=3
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: should this not be `${testsToSkip}` ?

**Comment by FRidh** [comment]
@domenkozar that seems to be an unrelated issue with the new matplotlib 1.5.0.
`test_sql` is indeed failing on Python 3.5. 
I was able to build with these changes. Can you merge?

**Pull request merged**